src/services/translation.js

```diff
@@ -1,20 +1,16 @@
 import i18n from 'i18next';
+import i18NextHttpBackend from 'i18next-http-backend';
+import {initReactI18next} from 'react-i18next';
 
-export const init = ({lang}) => {
-    return new Promise(async (resolve, reject) => {
-        const response = await fetch(`/locales/${lang}/strings.json`);
```

Comment:
Fetch is being done by i18NextHttpBackend, and no longer needed in this custom solution